### ISSUE TITLE
Microphone Package/Library Isn't Supported on Windows

### ISSUE THREAD

**Issue by hasanshahid5678** [deepgram/deepgram-python-sdk#425]
## What is the current behavior?
I am trying to run the live streaming examples for async_http and async_microphone. It only prints 'Could not open socket:' and exits. I have also run synchronous code and it is running as expected.

## Steps to reproduce
run python examples/streaming/async_http/main.py

## Expected behavior
Could not open socket:

## Please tell us about your environment
Windows 11, Python 3.11.5

**Comment by dvonthenen** [comment]
Hi @hasanshahid5678 

Unfortunately, the Microphone package is not supported on Windows:
https://github.com/deepgram/deepgram-python-sdk/tree/main/examples/streaming/microphone

It's on the list to look at and I believe it's mostly a configuration issue. Just need to dedicate time to it.

**Comment by hasanshahid5678** [comment]
And I was only able to run aync_http using wsl terminal.

**Comment by dvonthenen** [comment]
Yes, all of the async examples include code specific to *nix operating systems to demonstrate exiting the application gracefully from a signal termination.If you remove that part of the example, they should run in windows.

All of the threaded examples don't require that OS dependent termination. If you can run the threaded clients, they can be using in async classes/functions/code and they also are also better for IO-performant applications like websocket.

**Comment by hasanshahid5678** [comment]
If I want to send a message through web sockets to the web, can I still use threaded example. If yes, please tell me how because await websocket.send() can only be used under async functions

**Comment by dvonthenen** [comment]
You can use both. If you want to use the async example, just remove the *nix specific signal handling.
